server.js

```diff
@@ -11,6 +11,9 @@ const nunjucks = require('nunjucks')
 const session = require('express-session')
 const cookieParser = require('cookie-parser')
 
+// Environment variables
+dotenv.config()
```

Comment:
To avoid this being moved after the requires again, can we add a comment here along the lines of 'load environment variables before defining routes'?